src/web/lib/components/ThemeColorsEditor/index.js

```diff
@@ -88,14 +104,22 @@ class ThemeColorsEditor extends React.Component {
                   )}
                   onClick={ev => this.handleClick(ev, name)}
                 >
-                  <span
-                    className="theme-unit__swatch"
-                    style={{ backgroundColor: colorToCSS(color) }}
-                    title={colorLabels[name]}
-                  />
-                  <span className="theme-unit__label" title={colorLabels[name]}>
-                    {colorLabels[name]}
+                  {color ?
+                    <span
+                      className="theme-unit__swatch"
+                      style={{backgroundColor: colorToCSS(color)}}
+                      title={labels[name]}
+                    />
+                    : <span
+                      className="theme-unit__default"
```

Comment:
This element and its CSS are now basically copies of the swatch, except less accessibility-friendly (the focus ring is not visible when focused).

Revert this change, and modify the logic in the original swatch element to render the dashed appearance.